person.rb

```diff
@@ -8,6 +10,7 @@ def initialize(id:, age:, name: 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
```

Comment:
- [ ]  As part of the requirements, you are supposed to create a has-many relationship side of Person and Rental. So kindly add a method to add rentals in the person Class